app/addClothe.tsx

```diff
@@ -1,59 +1,242 @@
+        clothe.sleeveSize = selectedSize;
+        
+        
+        // Add the updated clothe object to the database
         const success = await DatabaseController.addClothe(clothe);
-
+        
+        console.log("SELECTED TYPE: " + selectedType)
         if (success) {
-            console.log("Successfully added clothe!")
+            console.log("Successfully added clothe of name " + clothe.name + " of type " + clothe.type + " of color " + clothe.color + " of sleeve size " + clothe.sleeveSize + " with link " + clothe.link)
         } else {
             console.log("Failed to add clothe!")
         }
-
+    
         navigation.goBack();
     }
 
     return (
         <View style={styles.container}>
-            <TextInput style={styles.input} placeholder="Clothe name"
-                       onChangeText={(text) => clothe.name = text}/>
-            <View style={styles.separator} lightColor="#eee" darkColor="rgba(255,255,255,0.1)"/>
+            <View style={styles.photoContainer}>
+                <Image style={styles.image} source={{ uri: selectedLink }} /> 
+                {/*The URL here should be the URI from the photo capture feature. Simply make the selectedLink variable be the URI and that should make the class get that data*/}
+                <View style={styles.greyBottom}/>
+                <TextInput style={styles.input} placeholder="Type Item Name Here"
+                    onChangeText={(text) => setSelectedName(text)} // Update the selectedName state
+                    />       
+            </View>
+
+            <View style={styles.filterSquare}>
+                <Text style={styles.filterText}>Tags</Text>
+                <View style={styles.pickerBox}>
 
-            <TextInput style={styles.input} placeholder="Clothe description"
-                       onChangeText={(text) => clothe.description = text}/>
-            <View style={styles.separator} lightColor="#eee" darkColor="rgba(255,255,255,0.1)"/>
+                    <Picker style={styles.picker} itemStyle={styles.pick}
+                        selectedValue={selectedType} 
+                        onValueChange={(itemValue: Clothe.Type) => 
+                            setSelectedType(itemValue) // Update the selectedType state 
+                    }>
+                        <Picker.Item label="Type" value={Clothe.Type.Unknown} />
+                        <Picker.Item label="Shirt" value={Clothe.Type.Shirt} />
+                        <Picker.Item label="Pants" value={Clothe.Type.Pants} />
+                        <Picker.Item label="Shoes" value={Clothe.Type.Shoes} />
+                    </Picker>
 
-            <TextInput style={styles.input} placeholder="Clothe image link"
-                       onChangeText={(text) => clothe.link = text}/>
-            <View style={styles.separator} lightColor="#eee" darkColor="rgba(255,255,255,0.1)"/>
+                    <Picker style={styles.picker} itemStyle={styles.pick}
```

Comment:
So, I don't understand clearly what you are saying, but if what you are saying is:

"Let's go with pre-defined filter tags for less trouble in code, and use search to search for name" 

I say Yes, this is great. It's easier to maintain, and easier to implement, and it will still get the job done. Simpler code, less trouble, easier maintenance.

Reply:
I'm saying that instead of hardcoding each color option and sleeve size option, it might be better to extract them into an enum class.

Reply:
Sounds like a plan. If the only places where I need to change this are in these two files, I can pull it off in one commit. I'll make Enums based on the one you set up for colors, and sleeve sizes.

Reply:
OK, so I managed to change the type, color, and sleeve size to enums. The best part is that I did the pickers change dynamically to the the enums on the clothes. So, the possibility to make the tags adjustable is still possible. We just need to use the same code for the filter here. 

Hmm... now that I think of it I can use this code to implement the filter component. I could implement the filter component later and make it dynamic to the enums in the clothes class. Please take another look at the code.